website/docs/addon_applications_admin.md

```diff
@@ -60,4 +67,28 @@ In the image before you can see that we set most of the environment variables in
 In this example MTOA will automatically set the `MAYA_VERSION`(which is set by Maya Application environment) and `MTOA_VERSION` into the `MTOA` variable. We then use the `MTOA` to set all the other variables needed for it to function within Maya.
 ![tools](assets/settings/tools_01.png)
 
-All the tools defined in here can then be assigned to projects. You can also change the tools versions on any project level all the way down to individual asset or shot overrides. So it is possible to upgrade you render plugin for a single shot, while not risking the incompatibilities on the rest of the project.
\ No newline at end of file
+All the tools defined in here can then be assigned to projects. You can also change the tools versions on any project level all the way down to individual asset or shot overrides. So it is possible to upgrade you render plugin for a single shot, while not risking the incompatibilities on the rest of the project.
+
+## Applications filters
+Applications that will be shown in launcher tool based on values in this setting. It is profile based filtering that allows to filter applications for specific task types.
+
+Using default settings will show all applications for all tasks. That can be easily changed.
+
+Simple example:
+![settings_applications](assets/applications/admin_applications_profiles.png)
+
+In this case all tasks with `Modeling` type will show only `Maya 2025`. Any other task type will show `After Effects 2024`, `Maya 2025`, `Nuke 15.0` and `Photoshop 2024`.
+
+Note: It is recommended to have one profile without task types filter with standard set of applications used in studio.
+
+## Tools filters
+Tools that will be used on application launch. It is profile based filtering that allows to filter tools for a context. Tools can have more specific filtering than applications.
+
+## Applications and Tools attributes
+Settings based filtering was introduced in applications addon release 1.0.0 . Before that applications were set with `applications` attribute on project (located under project anatomy) and tools were set using `tools` attribute on Project, Folder and Task entities.
+
+We still keep this the attributes in the system. Newly introduced settings do allow to switch between new settings and attributes, so we don't break existing productions without option to change it back.
+
+By default, are used settings based filters, but that is automatically changed to use attributes when settings from older version of applications addon are converted, so it is not needed to change it manually.
+
+Please keep in mind that `applications` and `tools` attributes are deprecated, and they will be removed in future versions of applications addon.
```

Comment:
I think we should be much clearer about this being legacy and that there is OLD versus NEW behavior where the old behavior is deprecated.
```suggestion
## Legacy: Applications and Tools attributes

Settings based filtering was introduced in applications addon release 1.0.0 . Before that applications were set with `applications` attribute on project (located under project anatomy) and tools were set using `tools` attribute on Project, Folder and Task entities.

For backwards compatibility the settings do allow to switch between new settings and attributes and the legacy behavior, so we don't break existing productions without option to change it back.

:::note Legacy behavior
The legacy behavior is enabled by **disabling** the filters in settings:
```
ayon+settings://applications/project_applications/enabled
ayon+settings://applications/project_tools/enabled
```
:::

By default, the settings based filters are used, but that is automatically changed to use attributes when settings from older version of applications addon are converted, so it is not needed to change it manually for backwards compatibility from older addons.

:::warning Deprecated
Please keep in mind that `applications` and `tools` attributes are deprecated, and they will be removed in future versions of applications addon. It is recommended to use the 1.0.0+ application and tool filtering instead.
:::
```